### ISSUE TITLE
KNN curve code

### ISSUE THREAD

**Issue by kashif** [facebookresearch/moco-v3#5]
Thanks @endernewton for your work! I was wondering if you could kindly share your KNN classifier curve code somewhere, either here or in some other repo/gist?

Thanks again!
Kashif

**Comment by endernewton** [comment]
Yes this is planned. We will verify it on our side before publishing it. It may take a while though. Thanks for the patience!

**Comment by kashif** [comment]
thanks, @endernewton I will wait. BTW is it based on this from [45] in your paper: https://github.com/zhirongw/lemniscate.pytorch/blob/master/test.py#L67

**Comment by jiangzhengkai** [comment]
@endernewton Any progress for the KNN curve code?

**Comment by tangtaogo** [comment]
I had tested the given ViT-S model through the DINO KNN code,  but only got acc 32.5.  @endernewton

**Comment by jasperhyp** [comment]
+1

**Comment by amandaluof** [comment]
> I had tested the given ViT-S model through the DINO KNN code, but only got acc 32.5. @endernewton

I also use the DINO KNN code in my project and get very low top1 accuracy, about 5%. 

So do you think there is something wrong with Dino KNN ?
